packages/formation-react/src/components/Breadcrumbs/usage.stories.mdx

```diff
@@ -1,62 +0,0 @@
-import { Meta } from '@storybook/addon-docs/blocks';
-
-<Meta title="Library/Breadcrumbs/Usage" />
```

Comment:
Why was this deleted?

Reply:
I couldn't figure out the audience. It didn't seem useful for either engineers or designers, so I opted to remove it and see if anybody cared. :see_no_evil: